### PR TITLE
Prepare 2.14.1 release

### PR DESCRIPTION
**Pull request by rosen-vladimirov**
Set ios-sim-portable to working version and set version of appbuilder-cli to 2.14.1

Update README for 2.14.1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -357,8 +359,8 @@ If addressing the configuration issues does not resolve your problem, you can [r
 Features
 ===
 
-Latest version: Telerik AppBuilder 2.14 
-Release date: November 4, 2015
+Latest version: Telerik AppBuilder 2.14.1
+Release date: November 16, 2015
 
 > For a complete list of the improvements and updates available in Telerik AppBuilder 2.14, see <a href="http://docs.telerik.com/platform/appbuilder/release-notes/v2-14" target="_blank">Telerik AppBuilder 2.14 Release Notes</a>.
```
Comment (ikoevska) [inline README.md]: Copy paste from the previous instance :)

**Comment by rosen-vladimirov** [comment]
run ci

**Comment by ikoevska** [comment]
Sh(t)ip it =) :+1:

**Comment by Mitko-Kerezov** [comment]
:+1:

**Comment by teobugslayer** [comment]
:+1:

**Pull request merged**